test/configCases/asset-url/target-node2/webpack.config.js

```diff
@@ -0,0 +1,20 @@
+/** @type {import("../../../../").Configuration} */
+module.exports = {
+	mode: "development",
+	target: "node",
+	devtool: false,
+	output: {
+		assetModuleFilename: "[name][ext]"
+	},
+	module: {
+		rules: [
+			{
```

Comment:
These test cases should not have a rule for that. The default rule should cover that.